search/src/config.js

```diff
@@ -0,0 +1,15 @@
+let config = {}
+
+switch (process.env.NODE_ENV) {
+  case ('development'):
```

Comment:
Please require the correct file dynamically

Reply:
done, but files have to be in src, so the config files are under src/config/